.github/workflows/publish.yml

```
@@ -0,0 +1,29 @@
+name: Publish documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v2
+    - name: Restore Coursier cache
+      uses: coursier/cache-action@v3
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Set up Node.js and Yarn
+      uses: actions/setup-node@v2-beta
+      with:
+        node-version: 12
+    - name: Publish docs
+      run: sbt docs/docusaurusPublishGhpages
```

```suggestion
      run: amm scripts/releaseSite.sc
```

https://github.com/disneystreaming/weaver-test/blob/master/.drone.yml#L49

Also means we need to install ammonite here.